dataeng/jobs/analytics/AnswerDistribution.groovy

```diff
@@ -24,11 +24,10 @@ class AnswerDistribution {
                 wrappers common_wrappers(allVars)
                 publishers common_publishers(allVars)
                 steps {
+                    shell(dslFactory.readFileFromWorkspace('dataeng/resources/opsgenie-enable-heartbeat.sh'))
                     shell(dslFactory.readFileFromWorkspace('dataeng/resources/answer-distribution.sh'))
-                    if (env_config.get('OPSGENIE_HEARTBEAT_NAME') && allVars.get('OPSGENIE_HEARTBEAT_KEY')){
-                        shell("curl -X GET 'https://api.opsgenie.com/v2/heartbeats/" + env_config.get('OPSGENIE_HEARTBEAT_NAME') + "/ping' -H 'Authorization: GenieKey " + allVars.get('OPSGENIE_HEARTBEAT_KEY') + "'")
-                    }
-               }
+                    shell(dslFactory.readFileFromWorkspace('dataeng/resources/opsgenie-disable-heartbeat.sh'))
```

Comment:
Will these multiple `shell()` commands work? Will Jenkins simply glom these scripts together?

Reply:
jenkins will invoke them one after the other as completely separate scripts, so if you want to persist anything between then you'll have to write to disk (under $WORKSPACE).

Reply:
In this case, the enable and disable scripts are only making network calls so it should be fine.